foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-deploy/src/main/resources/templates/remote-auth-secret.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  namespace: argocd
+  namespace: ${parentArtifactId}
   name: remote-auth-config
```

Comment:
Do we want to change the name `remote-auth-config` so it doesn't clash with our secret created by the s3 local chart:
https://github.com/boozallen/aissemble/blob/dev/extensions/extensions-helm/aissemble-localstack-chart/templates/credential-secret.yaml

Reply:
Think we also need to create a argocd template for s3 local so that it gets deployed to argocd when doing local deployments